numpyro/infer/mcmc.py

```diff
@@ -239,6 +239,31 @@ class MCMC(object):
         on a same sized but different dataset will not result in additional compilation cost.
         Note that currently, this does not take effect for the case ``num_chains > 1``
         and ``chain_method == 'parallel'``.
+
+    .. note:: It is possible to mix parallel and vectorized sampling, i.e., run vectorized chains
+        on multiple devices using explicit `pmap`. Currently, doing so requires disabling the
+        progress bar. For example,
+
+        .. code-block:: python
```

Comment:
I think you need an empty line after this

Reply:
Thanks, just fixed that